.env.example

```
@@ -1,6 +1,9 @@
 # Go to -> https://docs.trustlesswork.com/trustless-work/developer-resources/request-api-key
 NEXT_PUBLIC_API_KEY=""
 
+# Crossmint demo (optional, required for /crossmint Crossmint mode)
+NEXT_PUBLIC_CROSSMINT_API_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)